### PR TITLE
Add outline to assignments view

### PR DESCRIPTION
**Pull request by heraldofsolace**
This commit adds the outline to the assignments view (#18 ). Also parses the markdown to show the outline in rich text 🌚 
Screenshot_20220122_115200

### CODE REVIEW

**Comment by heraldofsolace** [comment]
@karllhughes @KSiig Let me know if things work or not

File: components/assignments/single-assignment.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ import RequestAssignment from "../../components/assignments/request-assignment";
 import {assignmentStatuses} from "../../constants/assignment-statuses";
 import SubmitAssignment from "../../components/assignments/submit-assignment";
 import AssignmentHeader from "../../components/assignments/assignment-header";
-
+// import ReactMarkdown from 'react-markdown/react-markdown.min';
```
Comment (karllhughes) [inline components/assignments/single-assignment.js:10]: Do you want to leave this commented line in?

**Review by karllhughes — COMMENTED** [review]
Just one quick question/change @heraldofsolace, otherwise, this looks fine to me.

File: components/assignments/single-assignment.js (hunk 1 above)
Comment (heraldofsolace) [reply components/assignments/single-assignment.js]: Fixed

**Comment by KSiig** [comment]
@heraldofsolace Looks good to me.

**Comment by karllhughes** [comment]
Merged

**Pull request merged**